### PR TITLE
[BugFix] Return script execution output as ResultSet

### LINKED ISSUE #23044: Script execution result not showing in GUI mysql client
### Real behavior (Required)
Script execution result only shows in MySQL client cli, not showing in GUI client(GUI client only shows resultset, not showing infoMessage

### StarRocks version (Required)
 - You can get the StarRocks version by executing SQL `select current_version()`

### PR DESCRIPTION
**Pull request by decster**
## What type of PR is this:
- [x] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes:

Fixes #23044

## Problem Summary(Required):

Script execution result only shows in MySQL client cli, not showing in GUI client(GUI client only shows resultset, not showing info message. This PR splits script output to lines and makes those as ResultSet, so GUI clients cant show them correctly.

## Checklist:

- [x] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto backported to target branch
  - [x] 3.0
  - [x] 2.5
  - [ ] 2.4
  - [ ] 2.3

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 13 / 23 (56.52%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/qe/StmtExecutor.java|4|11|36.36%|[286, 287, 468, 1004, 1007, 1008, 1009]|
|:large_blue_circle:|com/starrocks/qe/ExecuteScriptExecutor.java|9|12|75.00%|[57, 101, 102]|

File: fe/fe-core/src/test/java/com/starrocks/pseudocluster/PseudoBackend.java [hunk 1]
```diff
@@ -999,7 +999,18 @@ public Future<PMVMaintenanceTaskResult> submitMVMaintenanceTaskAsync(PMVMaintena
 
         @Override
         public Future<ExecuteCommandResultPB> executeCommandAsync(ExecuteCommandRequestPB request) {
-            throw new org.apache.commons.lang3.NotImplementedException("TODO");
+            ExecuteCommandResultPB result = new ExecuteCommandResultPB();
+            StatusPB pStatus = new StatusPB();
+            pStatus.statusCode = 0;
+            result.status = pStatus;
+            if (request.command.equals("execute_script")) {
+                result.result = "dummy result";
```
Comment (chaoyli) [inline fe/fe-core/src/test/java/com/starrocks/pseudocluster/PseudoBackend.java:1007]: `result.result = "Empty set"` like 
image

**Review by chaoyli — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:heart_eyes: **pass** : 15 / 18 (83.33%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/qe/StmtExecutor.java|3|6|50.00%|[1000, 1001, 1002]|
|:large_blue_circle:|com/starrocks/qe/ExecuteScriptExecutor.java|12|12|100.00%|[]|

**Review by trueeyu — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-3.0

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-2.5

**Pull request merged**